README.md

```diff
@@ -1,2 +1,24 @@
-# hw01
-Repository for homework 1
+# Self-Introduction
+![](https://artificery.com/wp-content/uploads/2017/09/portland-oregon.jpg)
+
+Hi all, my name is Moiz and I'm a graduate student in the MAPSS program. I'm *really* new to coding, but here's my attempt at listing some basic info about myself:
+
+1. Concentration: Political Science
+2. Hometown: Portland, Oregon
+3. Undergraduate Education: Vassar College
+
+# Description of Changes into README.md
+
+1. For each **header**, I started the line with a hashtag symbol "#" and added a space before typing out the text of the header.Then, I added a line break below before starting the text of the section.
+2. To embed the image of Portland, I started the line with an exclamation point "!", typed two square brackets "{}" without text in-between because I did not want the image labeled, and placed the link of the photo within regular brackets "()".
+3. To place emphasis on the word "really" using italics, I placed the word between two stars "*".
+4. To create the list of basic info about myself, I simply typed out the list as shown above. A line break was added before and after the list.
+5. After completely writing out the text, I staged the "README.md" file and clicked the "commit" button.
+6. Next, I added a comment to commit and clicked the "push" button to sync the file with GitHub.
+7. Finally, on the GitHub page, I created a pull request on the original repository. In the "Branch" menu, I chose the branch containing all of my commits. Then, I clicked on the "New pull request" button to the right of the Branch menu. Next, I used the "base" branch dropdown menu to select the original branch and the "compare" branch dropdown menu to select the branch with my changes. Finally, I typed a title and description where prompted and clicked the "Create pull request" button.
+
+# Reflections on GitHub Workflow and Experience with Markdown
+
+1. It was not initially intuitive how and where to use Markdown syntax until I saw the syntax dropdown menu, with the option of writing in Markdown syntax, in the bottom right corner of the pane where the changes to the README.md file were later made. After I saw this, the process of creating changes was straightforward.
+2. The GitHub workflow seems to make good sense. Fork, clone, do homework and commit, push, and finally pull. Initially, I was confused as to where I was supposed to do the homework on RStudio. Again, after seeing the syntax dropdown menu, the tasks I needed to complete became easier to understand.
+3. With longer homework assignments, I understand the value of consistently committing in order to save work and  refer to prior changes down the line.
```

Comment:
Evaluation HW-xx for Jane Doe
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     Yes      |              |            |
| **Coding strategy**         |       Yes    |              |            |
| **Presentation**            |      Yes     |              |            |
| **Achievement, creativity** |   Yes        |              |            |
| **Ease of access**          |         Yes  |              |            |

Remarks:

* Elaborate on above, especially for "needs work."
Moiz did a great overall job -- I don't think anything needs work in this assignment. 

* Some specific praise?
I liked that Moiz's entire document was so well structured, largely in thanks to his use of the header syntax. 

* Something I learned?
Its really easy to insert beautiful images into markdown documents with links, without having to worry too much about the files being saved locally. 

* Specific constructive criticism?
None
* Something I know and that you, my peer, might like to know because it is relevant to something you struggled with.
None